klimaat_helpdesk/templates/core/includes/footer_text.html

```diff
@@ -9,7 +9,7 @@
 
     <div class="footer-bar__links">
       <span class="footer-bar__title">Een initiatief van
-        <a class="footer-bar__title-link" href=https://scientists4future.nl/">scientists for future</a> &
+        <a class="footer-bar__title-link" href=https://scientists4future.nl/">Scientists4Future NL</a> &
```

Comment:
This link is badly formatted (missing opening ")

Reply:
fixed in the new pull request